org.eclipse.lsp4e/plugin.properties

```diff
@@ -33,4 +33,5 @@ command.selection.range.up.name=Enclosing Element
 command.selection.range.up.description=Expand Selection To Enclosing Element
 command.selection.range.down.name=Restore To Last Selection
 command.selection.range.down.description=Expand Selection To Restore To Last Selection
-markers.name = Language Servers
\ No newline at end of file
+markers.name = Language Servers
+typeHierarchy.command.name = Show Type Hierarchy
```

Comment:
Wouldn't it be better to call it `Open Type Hierarchy` instead of `Show Type Hierarchy` for consistency (see `Open Call Hiererachy`)